src/main/java/mod/gate/utils/Tablist.java

```diff
@@ -49,10 +50,20 @@ public static String getArea() {
     }
     //endregion
 
+    //region Lines
+    private static ArrayList<String> Lines;
+
+    public static ArrayList<String> getLines() {
+        loadTablist();
+        return Lines;
+    }
+    //endregion
+
     private static void loadTablist() {
-        if (!Scoreboard.isOnSkyblock()) return;
+        if (!Scoreboard.isOnSkyblock()) return;//there's a small issue here
```

Comment:
What's the issue? And weird comments again

Reply:
was an issue when I started out as it loaded the tablist before it was accessible